model_info/enwiki.drafttopic.md

```diff
@@ -15,611 +15,947 @@ Model Information:
+		Geography.Regions.Africa.Africa*             0.995
+		Geography.Regions.Africa.Central Africa      0.999
+		Geography.Regions.Africa.Eastern Africa      0.999
+		Geography.Regions.Africa.Northern Africa     0.998
+		Geography.Regions.Africa.Southern Africa     0.999
+		Geography.Regions.Africa.Western Africa      0.999
+		Geography.Regions.Americas.Central America   0.998
+		Geography.Regions.Americas.North America     0.983
+		Geography.Regions.Americas.South America     0.997
+		Geography.Regions.Asia.Asia*                 0.98
+		Geography.Regions.Asia.Central Asia          0.999
+		Geography.Regions.Asia.East Asia             0.995
+		Geography.Regions.Asia.North Asia            0.997
+		Geography.Regions.Asia.South Asia            0.998
+		Geography.Regions.Asia.Southeast Asia        0.998
+		Geography.Regions.Asia.West Asia             0.998
+		Geography.Regions.Europe.Eastern Europe      0.994
+		Geography.Regions.Europe.Europe*             0.964
+		Geography.Regions.Europe.Northern Europe     0.992
+		Geography.Regions.Europe.Southern Europe     0.996
+		Geography.Regions.Europe.Western Europe      0.993
+		Geography.Regions.Oceania                    0.998
+		History and Society.Business and economics   0.991
+		History and Society.Education                0.995
+		History and Society.History                  0.995
+		History and Society.Military and warfare     0.993
+		History and Society.Politics and government  0.992
+		History and Society.Society                  0.995
+		History and Society.Transportation           0.995
 		STEM.Biology                                 0.997
-		History_And_Society.Transportation           0.996
-		STEM.Meteorology                             0.999
-		History_And_Society.Politics and government  0.994
-		Culture.Internet culture                     0.997
-		History_And_Society.Military and warfare     0.995
-		Culture.Media                                0.997
-		STEM.Mathematics                             0.999
+		STEM.Chemistry                               0.997
+		STEM.Computing                               0.993
+		STEM.Earth and environment                   0.998
+		STEM.Engineering                             0.997
+		STEM.Libraries & Information                 0.999
+		STEM.Mathematics                             0.998
+		STEM.Medicine & Health                       0.997
+		STEM.Physics                                 0.998
+		STEM.STEM*                                   0.968
 		STEM.Space                                   0.999
-		Culture.Performing arts                      0.995
-		Geography.Bodies of water                    0.998
-		Geography.Europe                             0.963
-		History_And_Society.History and society      0.993
-		Culture.Entertainment                        0.993
+		STEM.Technology                              0.989
 		-------------------------------------------  -----
-	precision (micro=0.826, macro=0.811):
+	precision (micro=0.628, macro=0.478):
```

Comment:
precision seems to have dropped a bit, but this was to be expected right?

Reply:
Yes.  It now accounts for population rates and that punishes precision and accuracy-like metrics.